src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -46,6 +50,11 @@
     private static final Pattern CONTEXTUAL_COMMAND_FORMAT = Pattern
             .compile("(?<prefixTab>\\S+) /(?<tab>\\S+) (?<commandWord>\\S+)(?<arguments>.*)");
 
+    private static final String TABS_ALL_EXCEPT_DETAILS = "Dashboard, Contacts, Schedule or Todos";
```

Comment:
Given that we are at v1.4 where we are freezing features, I think this is acceptable. However, if we were to expand functionalities in the future, we should assign a story point to generate the available tabs.